src/ValoremOptionsClearinghouse.sol

```diff
@@ -180,7 +180,7 @@ contract ValoremOptionsClearinghouse is ERC1155, IValoremOptionsClearinghouse {
     }
 
     /// @inheritdoc IValoremOptionsClearinghouse
-    function claim(uint256 claimId) external view returns (Claim memory claimInfo) {
```

Comment:
We cannot/should not change this interface.

Reply:
The interface doesn't change, it is still external in the interface.

Reply:
Depending how much optimization we do on Claim accounting in redeem, we may not need to call this fn from inside the contract.